nearcore/src/runtime/mod.rs

```diff
@@ -325,8 +224,12 @@ impl NightshadeRuntime {
                 .expect("Store failed on genesis intialization")
                 .expect("Genesis state roots not found in storage")
         } else {
+            let runtime_config_store = Self::create_runtime_config_store(&genesis.config.chain_id);
+            let runtime_config = runtime_config_store.get_config(genesis.config.protocol_version);
+            let store_usage_config = &runtime_config.fees.storage_usage_config;
```

Comment:
You don't need to create runtime config store again if you pass correct config from `NightshadeRuntime::new`

Reply:
This is a static function, not a member function